bitnami/argo-workflows/values.yaml

```diff
@@ -672,10 +672,9 @@ controller:
           name: {{ include "argo-workflows.controller.database.password.secret" . }}
           key: {{ include "argo-workflows.controller.database.password.secret.key" . }}
     {{- end }}
-    ## Uncomment to set workflowDefaults
-    ## Ref: https://argoproj.github.io/argo-workflows/default-workflow-specs/#setting-default-workflow-values
-    ## workflowDefaults: {}
-    ##
+    {{- if .Values.controller.workflowDefaults }}
+    workflowDefaults: {{- include "common.tplvalues.render" (dict "value" .Values.controller.workflowDefaults "context" $) | nindent 2 }}
```

Comment:
Bear in mind that the entire `config` file is indented using a value of 4, you may need to update the value here too:

```diff
- workflowDefaults: {{- include "common.tplvalues.render" (dict "value" .Values.controller.workflowDefaults "context" $) | nindent 2 }}
+ workflowDefaults: {{- include "common.tplvalues.render" (dict "value" .Values.controller.workflowDefaults "context" $) | nindent 4 }}
```

Reply:
>Indent should be counted relative to the config file level and each nested level should add 2 spaces (which is the convention throughout values.yaml).

You are absolutely right! I missed the point that the param itself is also adding indentation. 

> Because of that, I claim that metricsConfig, telemetryConfig, sso and workflowDefaults should all be using indent 2.

Would you mind adding the changes to fix the level of indentation as well in the PR?

Reply:
Added the indent changes